canonical-customers.md

```diff
@@ -0,0 +1,38 @@
+# Canonical Customer Scenarios
+
+To provide a common reference across the SCITT documents and use cases, we'll use a few fictitious software and hardware companies.
+
+## Wabbit Networks
+
+A small software company, producing network monitoring software. Wabbit Networks is likely not known by most customers, and therefore needs some verifiable identity and claims to assure customers can trust their software.
+
+### Wabbit Networks Products
+
+Wabbit Networks produces several software products, including:
+
+- **Network Monitor**: network monitoring software that customers consume and run on their networks
```

Comment:
```suggestion
- **Network Monitor**: A network monitoring software that customers consume that run on their networks
```

Reply:
Thanks @yogeshbdeshpande, I see the edit you were shooting for and re-worded a bit. Please re-open if it didn't hit the mark